src/pages/Chat.jsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState, useRef } from 'react';
+import styled from '@emotion/styled';
+import ChatHeader from '../components/Chat/ChatHeader';
+import ChatBody from '../components/Chat/ChatBody';
+import ChatFooter from '../components/Chat/ChatFooter';
+
+export default function Chat() {
+  const [messages, setMessages] = useState([]);
+  const [users, setUsers] = useState({});
+  const chatBodyRef = useRef(null);
+
+  const getMessages = async () => {
+    const res = await fetch('/data/messages.json');
+    const data = await res.json();
+    setMessages(data.messages);
+  };
+
+  const getUsers = async () => {
+    const res = await fetch('/data/users.json');
+    const data = await res.json();
+    setUsers(data.users);
+  };
+
+  const handleSendMessage = async (content) => {
+    const newMessage = {
+      userId: 'hijh_0522',
+      isSender: true,
+      content,
+      timestamp: new Date().toISOString(),
+    };
+
+    const updatedMessages = [...messages, newMessage];
+    setMessages(updatedMessages);
```

Comment:
이렇게 간단히 써 볼 수도 있을 것 같아요!
```suggestion
    setMessages((prev) => [...prev, newMessage]);
```

Reply:
이 부분도 참고하여 리팩토링 하겠습니다! 👍